P5/Source/Specs/att.datcat.xml

```diff
@@ -219,6 +270,52 @@ $Id$
       its values, which are used as direct references to data categories; hence the use of
         <att>datcat</att> in the <gi>symbol</gi> element.</p>
   </exemplum>
+  <exemplum xml:lang="en">
+    <p>The <ident type="class">att.datcat</ident> attributes can be used for any sort of taxonomies.
+      The example below illustrates their usefulness for describing usage domain labels in
+      dictionaries on the example of the <hi rend="it">Diccionario da Lingua Portugueza</hi> by
```

Comment:
@bansp This `<hi rend='it`>` should probably be `<title level='m'>`, yes?

Reply:
By all means. Thanks, Elisa and Hugh, for having a look at it. Gonna change the markup right away,